Library/Formula/llvm.rb

```diff
@@ -78,17 +92,25 @@ class Llvm < Formula
       url "http://llvm.org/git/lldb.git"
     end
 
-    resource "clang-tools-extra" do
-      url "http://llvm.org/git/clang-tools-extra.git"
+    resource "polly" do
+      url "http://llvm.org/git/polly.git"
+    end
+
+    resource "sanitizers" do
+      url "http://llvm.org/git/compiler-rt.git"
     end
   end
 
   option :universal
-  option "with-clang", "Build Clang support library"
+  option "with-clang", "Build the Clang compiler and support libraries"
+  option "with-extra-tools", "Build extra tools for Clang"
+  option "with-libcxx", "Build the libc++ standard library"
```

Comment:
Shouldn't we build libc++ unconditionally for clang support? Without it, clang++ will be broken or useless, right?